test/e2e/delete_test.go

```diff
@@ -74,4 +74,7 @@ func testWindowsNodeDeletion(t *testing.T) {
 	err = framework.Global.KubeClient.CoreV1().Secrets("openshift-windows-machine-config-operator").Delete(context.TODO(), secrets.PrivateKeySecret, meta.DeleteOptions{})
 	require.NoError(t, err, "could not delete privateKey secret")
 
+	// Cleanup wmco-test namespace created by us.
+	err = testCtx.deleteNamespace(testCtx.workloadNamespace)
+	require.NoError(t, err, "could not delete test namespace")
```

Comment:
Why is this change part of this PR? Question implies that it the **why** needs to be called out in the commit message.

Reply:
@saifshaikh48 please comment on this. Can we split this out into a separate PR, or will it break things here?

Reply:
It is needed in this PR as the new metrics e2e test relies on the wmco-test namespace to deploy pods. Instead of deleting this namespace after networking tests complete, we want to allow it to persist throughout testing (this would be useful for tests added in the future as well).

Reply:
@saifshaikh48 Looks like [this](https://github.com/openshift/windows-machine-config-operator/pull/148/commits/e2057970f2bc29e2854afb5d941d5fc51f40a880#diff-cd50fad6da0839c40258ca356bf995a7ee5e1d5c270ee0af2cc5752e2f10a454R72) commit is also handling the same change.

Reply:
@VaishnaviHire what do we usually do in such scenarios?